pkg/api/dependencies.go

```diff
@@ -36,7 +36,7 @@ type Assembler interface {
 	AssembleProposal(metadata []byte, requests [][]byte) bft.Proposal
 }
 
-// WriteAheadLog is a write ahead log.
+// WriteAheadLog is wroted ahead log.
```

Comment:
```suggestion
// WriteAheadLog is write ahead log.
```

the previous comment was correct

Reply:
corrected